RecoBTag/TensorFlow/src/JetConverter.cc

```diff
@@ -1,25 +1,19 @@
-#ifndef RecoBTag_DeepFlavour_JetConverter_h
-#define RecoBTag_DeepFlavour_JetConverter_h
-
+#include "RecoBTag/TensorFlow/interface/JetConverter.h"
 #include "DataFormats/BTauReco/interface/JetFeatures.h"
 
 #include "DataFormats/JetReco/interface/Jet.h"
 
 namespace btagbtvdeep {
+  
 
-  class JetConverter {
-    public:
-
-      static void JetToFeatures(const reco::Jet & jet,
-                                JetFeatures & jet_features) {
+  static void jetToFeatures(const reco::Jet & jet,
+			    JetFeatures & jet_features) {
         jet_features.pt = jet.pt(); // uncorrected
```

Comment:
this function is already defined in JetConverter.h and the compiler generates a warning

Reply:
I think that either the implementation needs to be removed in .h, or this .cc file should just simply include .h. The latter seems better to me.

Reply:
@slava77 thanks for the suggestion. So it's ok to have `JetConverter.cc` be just one line?
```
#include "DataFormats/BTauReco/interface/JetFeatures.h"
```
thanks

Reply:
yes, AFAIK this works